src/core_plugins/kibana/public/management/sections/indices/edit_index_pattern/indexed_fields_table/components/table/table.js

```diff
@@ -25,9 +25,12 @@ export class Table extends PureComponent {
         {isTimeField ? (
           <span>
             &nbsp;
-            <EuiToolTip content="This field represents the time that events occurred.">
-              <EuiIcon type="clock" color="primary" />
-            </EuiToolTip>
+            <EuiIconTip
+              type="clock"
+              color="primary"
+              aria-label="Info"
```

Comment:
I would rather suggest using a short version of the description here as a label, like "Primary time field".
That way a screenreader would read out:

:speaking_head: Primary time field *-pause-* This field represents the time that events occurred.
vs:
:speaking_head: Info *-pause-* This field represents the time that events occurred.

That way a screen reader user once familiar to what the time field means, can immediately jump to the next element, once she heard "primary time field", instead of needing to wait for the long explanation.

Reply:
makes sense! updating